package.json

```diff
@@ -68,7 +68,7 @@
     "eslint-plugin-n": "^15.7.0",
     "eslint-plugin-promise": "^6.1.1",
     "gl": "^6.0.2",
-    "jsdoc": "^3.6.11",
+    "jsdoc": "^4.0.2",
```

Comment:
This fixes a critical bug with jsdoc version `3.6.11`. I fixed this with `npm audit fix --force`.

Reply:
I already tried version 4 of jsdoc, but it fails with template I using for jsroot documentation generation.
Before one found a solution - one need to use version 3 :(